gitlab-templates/shared/build_and_test_acceptance.yml

```diff
@@ -0,0 +1,123 @@
+
```

Comment:
I suggest we remove the "templates" word from the dir name, as these are not templates IMO but just the bits of one big pipeline.

Maybe:
```
gitlab-pipeline/shared/...
gitlab-pipeline/stagee/...
```
or

```
gitlab-include/shared/...
gitlab-include/stagee/...
```

Reply:
Yes, agreed.

I like `gitlab-pipeline` the best I think.